podcasts/DiscoverViewController.swift

```diff
@@ -145,13 +145,29 @@ class DiscoverViewController: PCViewController {
         }
     }
 
+    /// Reloads discover, keeping the items listed in `exclude`
+    /// - Parameters:
+    ///   - items: Items to exclude from the reload process. These items will REMAIN in Discover
+    ///   - category: The `DiscoverCategory` to add to the layout. This is sort of an artifical `DiscoverLayout`.
+    func reload(except items: [DiscoverItem], category: DiscoverCategory) {
+        let categoryVC = CategoryPodcastsViewController(category: category)
+        categoryVC.registerDiscoverDelegate(self)
```

Comment:
[nit] Maybe we can just expose `delegate` and do the usual `categoryVC.delegate = ...`?

Reply:
Sounds good 👍 

I had initially conformed to `DiscoverSummaryProtocol` but not needed any more.